config/printer-Mingda-Magician-X-2023.cfg

```diff
@@ -0,0 +1,195 @@
+# This file contains common pin mappings for the Mingda Magician X
+# Magician X To use this config, the firmware should be compiled for the
+# STM32F407 with a "64KiB bootloader" and USB communication PA11/PA12 it will connect to the USB A port.
+#usart3 will connect to the esp01 header
+# The "make flash" command does not work on the Mingda. Instead,
+# after running "make", copy the generated "out/klipper.bin" file to a
+# file named "firmware.bin" on an SD card and then restart the Printer
+# with that SD card.
+
+# See docs/Config_Reference.md for a description of parameters.
+#stm32 f407 64kb bootloader and 8mhz clock usb a
+
+[include macros.cfg]
+[include mainsail.cfg]
```

Comment:
please do not include cfg files that an end user may not have or need

Thanks
James